SortApp/SortApp.ViewModel/MainWindowViewModel.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SortApp.BL;
+
+namespace SortApp.ViewModel
+{
+	/// <summary>
+	/// Represents the view model object for data.
+	/// </summary>
+	/// <owner>Oleh Petrenko</owner>
+	public sealed class MainWindowViewModel : ViewModelBase
+	{
+		/// <summary>
```

Comment:
Please, check if your VS settings for tabulation size if set by project guidelines. This tabs seems to large for me.